docs/querying/sql-window-functions.md

```diff
@@ -0,0 +1,164 @@
+
+The following example returns the rank of each row in ascending order based upon its `delta` value as related to other rows with the same `channel` value.
+
+```sql
+SELECT FLOOR(__time TO DAY) AS event_time,
+    channel,
+    ABS(delta) AS change,
+    RANK() OVER w AS rank_value
+FROM wikipedia
+WHERE channel in ('#kk.wikipedia', '#lt.wikipedia')
+AND '2016-06-28' > FLOOR(__time TO DAY) > '2016-06-26'
+GROUP BY channel, ABS(delta), __time
+WINDOW w AS (PARTITION BY channel ORDER BY abs(delta) ASC)
+```
+
+<details>
+<summary> View results </summary>
+
+| event_time | channel | change | rank_value |
+| -- | -- | -- | -- |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 1	| 1 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 1	| 1 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 7	| 3 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 56 | 4 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 56 | 4 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 63 | 6 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 91 | 7 |  
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 2440 | 8 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 2703 | 9 |
+| 2016-06-27T00:00:00.000Z	| #kk.wikipedia	| 6900 |10 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 1	| 1 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 2	| 2 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 13 | 3 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 28 | 4 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 53 | 5 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 56 | 6 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 59 | 7 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 391 | 8 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 894	| 9 |
+| 2016-06-27T00:00:00.000Z	| #lt.wikipedia	| 4358 | 10 |
+
+</details>
+
+Window functions are similar to [aggregation funtctions](./aggregations.md).  
+
+You can use the OVER clause treat other Druid aggretation functions as window functions. For example, the sum of a value for rows within a window.
+
+When working with window functions, consider the following:
+- Window functions only work on GROUP BY queries.
+- Window functions support aliasing.
+
+## Define a window with the OVER clause
+
+The OVER clause defines the query windows for window functions as follows:
+- PARTITION BY indicates the dimension that defines the rows within the window
+- ORDER BY specifies the order of the rows within the windows.
+Sometimes windows are called partitions, but don't confuse them with the segment partitioning feature of Druid.
+
+
+The PARTITIONED BY clause defines the dimenson to use for the window. For example, the following OVER clause sets the window dimension to `channel` and orders the results by the absolute value of `delta` ascending:
+
+```sql
+...
+rank() OVER (PARTITION BY channel ORDER BY ABS(delta) ASC)
```

Comment:
Add an explanation on why you would use this query instead of the one above, even though they are functionally the same. Because the previous one is a GROUP BY but this one can be used in other queries such as...?

Reply:
I don't understand. This is not a query, just an example of an OVER clause. If you wrote out the whole query, it would need to be a GROUP BY. The only difference here is that the window definition is inline (PARTITION.... ) instead of being aliased (w) in the previous example

Reply:
Oh I see that, never mind my comment.